### ISSUE TITLE
Five failures on openSUSE Python 2

### ISSUE THREAD

**Issue by jayvdb** [elmotec/massedit#15]
Packanging project at https://build.opensuse.org/package/show/home:jayvdb:py-new/python-massedit
```
[   64s] ======================================================================
[   64s] ERROR: test_exec_option (tests.TestCommandLine)
[   64s] Check trivial call using executable.
[   64s] ----------------------------------------------------------------------
[   64s] Traceback (most recent call last):
[   64s]   File "/home/abuild/rpmbuild/BUILD/massedit-0.68.5/tests.py", line 714, in test_exec_option
[   64s]     self.assertEqual(actual[3], '-#!/usr/bin/env python')
[   64s] IndexError: list index out of range
[   64s] 
[   64s] ======================================================================
[   64s] ERROR: test_file_option (tests.TestCommandLine)
[   64s] Test processing of a file.
[   64s] ----------------------------------------------------------------------
[   64s] Traceback (most recent call last):
[   64s]   File "/home/abuild/rpmbuild/BUILD/massedit-0.68.5/tests.py", line 644, in test_file_option
[   64s]     actual = output.getvalue().split("\n")[3]
[   64s] IndexError: list index out of range
[   64s] 
[   64s] ======================================================================
[   64s] FAIL: test_error_in_function (tests.TestCommandLine)
[   64s] Check error when the function triggers an exception.
[   64s] ----------------------------------------------------------------------
[   64s] Traceback (most recent call last):
[   64s]   File "/home/abuild/rpmbuild/BUILD/massedit-0.68.5/tests.py", line 701, in test_error_in_function
[   64s]     output=output)
[   64s] AssertionError: ZeroDivisionError not raised
[   64s] 
[   64s] ======================================================================
[   64s] FAIL: test_missing_function_name (tests.TestCommandLine)
[   64s] Check error when the function is empty but not the module.
[   64s] ----------------------------------------------------------------------
[   64s] Traceback (most recent call last):
[   64s]   File "/home/abuild/rpmbuild/BUILD/massedit-0.68.5/tests.py", line 680, in test_missing_function_name
[   64s]     self.assertEqual(log_sink.log, expected)
[   64s] AssertionError: u'' != u"'massedit:' is not a callable function: 'dict' object has no attribute 'massed [truncated]...
[   64s] + 'massedit:' is not a callable function: 'dict' object has no attribute 'massedit'
[   64s] 
[   64s] 
[   64s] ======================================================================
[   64s] FAIL: test_wrong_number_of_argument (tests.TestCommandLine)
[   64s] Test passing function that has the wrong number of arguments.
[   64s] ----------------------------------------------------------------------
[   64s] Traceback (most recent call last):
[   64s]   File "/home/abuild/rpmbuild/BUILD/massedit-0.68.5/tests.py", line 690, in test_wrong_number_of_argument
[   64s]     self.assertEqual(log_sink.log, expected)
[   64s] AssertionError: u'' != u"'massedit:get_function' is not a callable function: function should take 2 arg [truncated]...
[   64s] + 'massedit:get_function' is not a callable function: function should take 2 arguments: lines, file_name
```

**Comment by povilasb** [comment]
Forget about python 2 support? :)

**Comment by jayvdb** [comment]
I dont mind if Python 2 support is removed.

**Comment by elmotec** [comment]
Right, Python 2.7 is no longer supported officially. 

I gave it a go none the less because I wanted to upgrade the CI workflow anyway but I could not reproduce the problem on [GitHub/Ubuntu 18.04](https://github.com/elmotec/massedit/runs/1598199448) or Appveyor. It's kind of a pain to install WSL Suse just to test python 2.7. 

I just pushed the exact version I tested to https://pypi.org/project/massedit/.
